content/cpp/concepts/stacks/terms/top/top.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.top()'
+Description: 'Returns the element on the top of the stack.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Objects'
+  - 'OOP'
+  - 'Classes'
+  - 'Stacks'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`.top()`** method returns the top element on the [`stack`](https://www.codecademy.com/resources/docs/cpp/stacks). This will be the most recently added element on the `stack`.
```

Comment:
```suggestion
The **`.top()`** method returns the top element on the [`stack`](https://www.codecademy.com/resources/docs/cpp/stacks). This will be the most recently added element on the `stack` since they follow a last-in first-out (LIFO) insertion order.
```